package.json

```diff
@@ -50,7 +50,7 @@
     "react-intl": "^2.1.2",
     "react-router": "^3.2.1",
     "serve-favicon": "^2.3.0",
-    "swagger-ui": "^3.17.3",
```

Comment:
Please don't pin a specific version: the exact constraints we know of are “over `3.17.3` and under `3.18`, which is `~3.17.3`” 🙂

> See more on the [tilde operator documentation](https://docs.npmjs.com/misc/semver#tilde-ranges-123-12-1).